### PR TITLE
added peo

### PR DESCRIPTION
**Pull request by simonegramegna**

### CODE REVIEW

File: peo/readme.md
Comment (dgarijo) [inline peo/readme.md:1]: Please complete the readme file

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please address the issues outlined in my review

File: peo/readme.md
Comment (simonegramegna) [reply peo/readme.md:1]: Hi, thanks for the response. I have no idea on how to publish and I need a tutorial or a help. The repository of my ontology is: https://github.com/simonegramegna/peo_ontology . Could you please help me?

Comment (dgarijo) [reply peo/readme.md:1]: See https://github.com/perma-id/w3id.org/tree/master/example for an example

Comment (simonegramegna) [reply peo/readme.md:1]: ok thank you, I'll check

**Comment by simonegramegna** [comment]
I did some changes, I don't know if the .htaccess is correct

File: peo/.htaccess [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Attiva il Rewrite Engine
+RewriteEngine On
+
+# Redirige richieste generiche all'ontologia RDF come default
+RewriteRule ^$ https://raw.githubusercontent.com/simonegramegna/peo_ontology/main/peo_ontology.rdf [R=302,L]
```
Comment (dgarijo) [inline peo/.htaccess:5]: If you leave this redirection first, then all the ones below will not work. I would remove line 5.

**Comment by dgarijo** [comment]
Please see the  pending comment. Thanks!

File: peo/.htaccess (hunk 1 above)
Comment (simonegramegna) [reply peo/.htaccess:5]: instead I remove the lines belows beacuse the main ontology is at that link, should this work?

Comment (dgarijo) [reply peo/.htaccess:5]: It will work, but you won't do content negotiation.
If you remove line 5, content negotiation will work for ttl and rdf/xml

**Comment by simonegramegna** [comment]
> Please see the pending comment. Thanks!

I fiexed, it should work

**Review by dgarijo — APPROVED** [review]

**Pull request merged**